tests/workflow/interfaces/execute/test_jax_jit.py

```diff
@@ -915,10 +915,7 @@ def test_jit_allcounts(self, device_name, counts_wires):
 
     @pytest.mark.parametrize(
         "device_name",
-        (
-            pytest.param("default.qubit", marks=pytest.mark.xfail),
-            pytest.param("reference.qubit", marks=pytest.mark.xfail),
-        ),
+        (pytest.param("default.qubit", marks=pytest.mark.xfail),),
```

Comment:
If only one parameter and supposed to fail currently do we still need to parametrize them? Would it be better to direct xfail along with fail reason short description

Reply:
✔️ Updated